application/app/src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,25 @@
+# This file is required for running the API via the make commands
```

Comment:
We shouldn't be creating Spring profiles as we use make profiles for that and drive everything from variables.